packages/manager/test/SliceMachineManager-versions-checkIsAdapterUpdateAvailable.test.ts

```diff
@@ -0,0 +1,64 @@
+import { expect, it } from "vitest";
+
+import { createTestPlugin } from "./__testutils__/createTestPlugin";
+import { createTestProject } from "./__testutils__/createTestProject";
+import { mockNPMRegistryAPI } from "./__testutils__/mockNPMRegistryAPI";
+import { mockAdapterUIDirectory } from "./__testutils__/mockAdapterUIDirectory";
+
+import { createSliceMachineManager } from "../src";
+
+it("returns true if an update is available", async (ctx) => {
+	const adapter = createTestPlugin();
+	const cwd = await createTestProject({ adapter });
+	const manager = createSliceMachineManager({
+		nativePlugins: { [adapter.meta.name]: adapter },
+		cwd,
+	});
+	const packageJSON = {
+		name: adapter.meta.name,
+		version: "1.0.0",
+	};
+
+	await mockAdapterUIDirectory({
```

Comment:
Similar to the previous comment, we can remove the "UI" part of the name.

Reply:
Done, thanks 🙂